### ISSUE TITLE
Insufficient color contrast 

### ISSUE THREAD

**Issue by kingalg** [TryQuiet/quiet#2592]
Hi @jgaylor I have two types of elements with insufficient color contrast. Those elements are present in more than one place, but I got the below examples from Google Console. If you figure out which color would still match the rest of our designs but have sufficient contrast, leave a comment here, and I'll create an issue with a list of places where we need to change it in our app.

The item's text contrast ratio is 2.09. This ratio is based on an estimated foreground color of #7DBDD0 and an estimated background color of #FFFFFF. Consider increasing this item's text contrast ratio to 4.50 or greater.
<img width="284" alt="Screenshot 2024-08-06 at 15 56 33" src="https://github.com/user-attachments/assets/1d956972-1dfe-4d75-8fad-4aece13276b6">

The item's text contrast ratio is 1.74. This ratio is based on an estimated foreground color of #C4C4C4 and an estimated background color of #FFFFFF. Consider increasing this item's text contrast ratio to 4.50 or greater. 
I know that this is a placeholder, but placeholder text must meet a color contrast requirement of at least 4.5:1 contrast ratio, or 3:1 contrast ratio if its large (larger than 18pt or 14pt + bold) to satisfy WCAG 2.0 AA requirements. To quote the [W3.org success criteria 1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html), "minimum contrast success criterion (1.4.3) applies to text in the page, including placeholder text".
<img width="284" alt="Screenshot 2024-08-06 at 15 57 40" src="https://github.com/user-attachments/assets/181ba029-5911-4927-ba10-6f1e4a3bced4">

**Comment by jgaylor** [comment]
@kingalg thanks for bringing this to my attention, and for the visuals. Really helpful info:

1. **Blue links:** Yes, this is deprecated. We have a darker blue (#1B6FEC) in our global Figma variables/tokens that we use for inline links such as these. It has a contrast of 4.63. Let's use this instead.
2. **Placeholder text in forms:** Perfect, I didn't catch this one, let's use #767676 for the placeholder text which is 4.54. I've added this to the global variables/tokens in Figma and set the placeholder text to use it in all forms.

**Comment by kingalg** [comment]
Thanks @jgaylor! 

I created an issue https://github.com/TryQuiet/quiet/issues/2593 with list of places where we still need to exchange those colors for new designs.
